pyglossary/plugins/ebook_mobi.py

```diff
@@ -93,13 +87,32 @@
 		comment="Include index page",
 	),
 	"apply_css": StrOption(
-		disabled=True,
+		# disabled=True,
 		comment="Path to css file",
 	),
 	"cover_path": StrOption(
-		disabled=True,
+		# disabled=True,
 		comment="Path to cover file",
 	),
+	"group_size": IntOption(
```

Comment:
Please rename to `max_file_size`, and change it from KB to bytes. Use `FileSizeOption` like in Aard2 plugin (aard2_slob.py).